kitchen.yml

```diff
@@ -31,8 +31,8 @@ platforms:
       privileged: true
       pid_one_command: /usr/lib/systemd/systemd
       intermediate_instructions:
-      - RUN yum install -y net-tools
-      - RUN yum install -y iptables # So crapy but manage bad dependencies of docker packages
+        - RUN yum install -y net-tools
```

Comment:
This should be on a single line to avoid another layer

Reply:
Please take over this one @tas50  so many of these to do.

Reply:
Done